ArduCopter/RC_Channel.cpp

```diff
@@ -142,6 +142,16 @@ void RC_Channel_Copter::do_aux_function_change_mode(const Mode::Number mode,
     }
 }
 
+void RC_Channel_Copter::do_aux_function_armdisarm(const aux_switch_pos_t ch_flag)
+{
+    RC_Channel::do_aux_function_armdisarm(ch_flag);
```

Comment:
if the vehicle was already armed I guess the armed-with-switch would become true?  I guess we don't need to check that we were disarmed before and armed after?